contributors/devel/sig-architecture/api_changes.md

```diff
@@ -744,6 +744,13 @@ doing!
 Check out the [E2E docs](../sig-testing/e2e-tests.md) for detailed information about how to
 write end-to-end tests for your feature.
 
+When introducing a new API or field as an alpha feature, you have to add a
+[Feature label](../sig-testing/e2e-tests.md#kinds-of-tests). This prevents the
+test from running in the main suite, which has the feature disabled by default.
```

Comment:
They should also explicitly opt-in that test like here: https://github.com/kubernetes/test-infra/pull/20833/files

Reply:
Thanks!